tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_bluetooth.py

```diff
@@ -27,15 +23,25 @@ def test_toggle_bluetooth_settings(self):
         settings = Settings(self.marionette)
         settings.launch()
 
+        self.assertFalse(self.data_layer.bluetooth_is_enabled)
+        self.assertFalse(self.data_layer.get_setting('bluetooth.visible'))
+
         bluetooth_settings = settings.open_bluetooth_settings()
         bluetooth_settings.enable_bluetooth()
 
+        self.assertTrue(self.data_layer.bluetooth_is_enabled)
+        self.assertTrue(self.data_layer.get_setting('bluetooth.visible'))
+
         bluetooth_settings.tap_rename_my_device()
         bluetooth_settings.type_phone_name(device_name)
         bluetooth_settings.tap_update_device_name_ok()
 
         bluetooth_settings.enable_visible_to_all()
 
-        # Now have host machine inquire and shouldn't find our device
-        device_found = self.bluetooth_host.is_device_visible(device_name)
-        self.assertTrue(device_found, "Host should see our device (device discoverable mode is ON)")
+        if 'bluetoothhost' in self.testvars:
+            # Bluetooth host object
+            from gaiatest.utils.bluetooth.bluetooth_host import BluetoothHost
+            self.bluetooth_host = BluetoothHost(self.marionette)
+            # Now have host machine inquire and shouldn't find our device
+            device_found = self.bluetooth_host.is_device_visible(device_name)
+            self.assertTrue(device_found, "Host should see our device (device discoverable mode is ON)")
```

Comment:
This looks okay, though I'd prefer `bluetooth_host` for the test variable key. I wonder if it's even worth preserving this test coverage though given it prevents anyone running the test without PyBluez (which only works on Linux) and we're unlikely to be able to keep it once we move to the remote lab.